doc/active_docs/account_management_api.json

```
@@ -3152,12 +3152,10 @@
             "content": {}
           }
         }
-      }
-    },
-    "/admin/api/account_plans/{account_plan_id}/features/{id}.xml": {
+      },
       "post": {
-        "summary": "Account Plan Features Create",
-        "description": "Associate an account feature to an account plan.",
+        "summary": "Account Plan Features Enable",
```

```suggestion
        "summary": "Account Plan Feature Enable",
```

Maybe this to clarify further that it is for a single feature.